R/agg_helpers.R

```diff
@@ -61,7 +61,7 @@ agg_categorical <- function(dt,
                  "* See `missing_dt_severity` argument if it is okay to only make ",
                  "aggregate/scale data that are possible given what is available.\n",
                  paste0(capture.output(missing_dt), collapse = "\n"))
-        assertive::assert_engine(empty_missing_dt, missing_dt,
+        assertive.base::assert_engine(empty_missing_dt, missing_dt,
```

Comment:
It looks like we need to come up with an alternative to signaling different severity levels before we can fully remove `assertive`.

Reply:
Yeah exactly. I got stuck on this but assertive.base seems to work for now.